Styling/README.md

```diff
@@ -598,6 +620,18 @@ Returns the arctangent value of the number (in radians).
 }
 ```
 
+#### atan2
+
+`atan2(GridY : Number, GridX : Number) : Number`
+
+Returns the arctangent of the quotient of `GridY` and `GridX` (in radians).
```

Comment:
For this one I meant to rename `Angle` and `temperature` to `GridY` and `GridX`, but the function definition/description can stay `y` `x`.